http-add-on/values.yaml

```diff
@@ -0,0 +1,14 @@
+version: v0.1.0-v1alpha1
+
+additionalLabels: ""
+
+crds:
+  install: true
+
+operator:
+  pullPolicy: Always
+
+images:
+  operator: ghcr.io/kedacore/http-add-on-operator
+  interceptor: ghcr.io/kedacore/http-add-on-interceptor
+  scaler: ghcr.io/kedacore/http-add-on-scaler
```

Comment:
nit, but can you explicitly add the `:latest` label here?

Reply:
I'd prefer not to do that as people should be able to pass that separately and it should use the Helm chart's app version by default

Reply:
See https://github.com/kedacore/charts/blob/master/keda/templates/12-keda-deployment.yaml#L50

Reply:
O added the chart version on the chart too, but we already had the `version` on the `Values.yaml`

Reply:
sounds good 👍